### ISSUE TITLE
[Feature] Vitals applet

### ISSUE THREAD

**Issue by N0m4n904** [pop-os/cosmic-epoch#658]
It would be nice to have some kind of vital applet to monitor temps and CPU/GPU usage. Just like the vitals gnome extension.

**Comment by leviport** [comment]
Duplicate of https://github.com/pop-os/cosmic-applets/issues/575

**Comment by authenticastrology** [comment]
Yes, but the one the link refers to doesn't handle temperature. For those of us on laptops, it is nice to be able to monitor that because a fan turning on doesn't really tell us anything.
